x/concentrated-liquidity/lp_test.go

```diff
@@ -295,6 +298,18 @@ func (s *KeeperTestSuite) TestCreatePosition() {
 
 			// Validate events emitted.
 			s.AssertEventEmitted(s.Ctx, types.TypeEvtCreatePosition, expectedNumCreatePositionEvents)
+
+			// Validate that listeners were called the desired number of times
+			expectedAfterInitialPoolPositionCreatedCallCount := 0
+			if !tc.isNotFirstPosition {
+				// We want the hook to be called only for the very first position in the pool.
+				// Such position initialized current sqrt price and tick. As a result,
+				// we want the hook to run for the purposes of creating twap records.
+				// On any subsequent update, adding liquidity does not change the price.
+				// Therefore, we do not have to call this hook.
+				expectedAfterInitialPoolPositionCreatedCallCount = 1
```

Comment:
lol, instead of the double negative, can we just set the expected to 1, and the if isNotFirstPosition then we expect 0

Reply:
Unsure what you mean here. The comment makes sense to me